bpf/headers/bpf_tracing.h

```diff
@@ -489,4 +489,11 @@ typeof(name(0)) name(struct pt_regs *ctx)				    \
 }									    \
 static __always_inline typeof(name(0)) ____##name(struct pt_regs *ctx, ##args)
 
+/* BPF_UPROBE and BPF_URETPROBE are identical to BPF_KPROBE and BPF_KRETPROBE,
+ * but are named way less confusingly for SEC("uprobe") and SEC("uretprobe")
+ * use cases.
+ */
+#define BPF_UPROBE(name, args...)  BPF_KPROBE(name, ##args)
```

Comment:
These helpers weren't in the file we had from Cilium, but they are in the newest libbpf version. I just copied them from there, helps with making the code clearer.